forwardbot_gitlab/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright <2023> <Vauxoo info@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Forwardbot for Gitlab",
+    "summary": "Automatically create forward ports",
+    "author": "Vauxoo",
+    "website": "https://github.com/Vauxoo/runbot-addons",
+    "license": "AGPL-3",
+    "category": "Bots",
+    "version": "11.0.1.0.0",
+    "depends": ["base"],
+    "external_dependencies": {"python": ["requests"]},
```

Comment:
You don't need to add external_dependencies where Odoo already have added:
 - https://github.com/odoo/odoo/blob/01717dc71bae261dad8ad6515ff2b1e2be861295/requirements.txt#L41

```suggestion
```